execution.py

```diff
@@ -1,502 +1,502 @@
-        else:
-            virtual_disk = data[0].split()
-            vd_name = virtual_disk[0]
-            workload = virtual_disk[1]
-            flag_run = 1
-            file = open(r"../../../Config/Test.txt", "w+")
-            for _ in data[1:]:
-                file.write(_)
-            file.close()
-        return vd_name, workload, flag_run
-    def execute_test(self):
-        '''
-        This method execute workload and create all required
-        configuration of SSY.
-        Arguments : None
-        Return: None
-        '''
-        vd_name, workload, flag_run  = self.input_for_test()
-        if flag_run == 1:
-            print('************************'\
-                  'Test Started************************\n')
-            LogCreat().logger_info.info('************************'\
-                                        'Test Started************************')
-            time.sleep(10)
-            if vd_name.lower() != "standard":
-                self.test_enable_cap_opt_at_server()
-            time.sleep(15)
-            self.create_diskpool(vd_name)
-            time.sleep(15)
-            self.stop_server()
-            time.sleep(25)
-            self.start_server()
-            time.sleep(25)
-            self.test_create_virtual_disk(vd_name)
-            time.sleep(50)
-            self.set_vd_properties(vd_name)
-            time.sleep(5)
-            self.test_serve_vd_to_host()
-            time.sleep(10)
-            diskindex = self.initialize_vd()
-            time.sleep(5)
-            print('************************'\
-                  'VdBench Execution Started************************\n')
-            LogCreat().logger_info.info('************************'\
-                                        'VdBench Execution Started************************')
-            VdBenchRun().run(vd_name, workload, diskindex)
-            print('Result creation completed')
-            print('************************'\
-                  'Setup Cleanup Started************************\n')
-            LogCreat().logger_info.info('************************'\
-                                        'Setup Cleanup Started************************')
-            self.un_server_vd()
-            time.sleep(25)
-            self.delete_vd()
-            time.sleep(180)
-            self.delete_pool()
-            time.sleep(180)
-            if vd_name.lower() != "standard":
-                self.test_disable_cap_opt_at_server()
-                time.sleep(30)
-            print('************************'\
-                  'VdBench Execution Completed************************\n')
-            LogCreat().logger_info.info('************************'\
-                                        'VdBench Execution Completed************************')
-    def verification(self,res_json, msg):
-        '''
-        This method used to log INFO and ERROR to log file.
-        Arguments (dict, str): res_json, msg
```

Comment:
Arguments should have one line definition

Reply:
Its really hard to give that much documentation there so please skip this comment